src/Xamarin.Android.Build.Tasks/Tasks/GenerateLayoutBindings.CSharpBindingGenerator.cs

```diff
@@ -157,8 +157,35 @@ void WriteClassClose (State state)
 				WriteLineIndent (state, "}");
 			}
 
+			void WriteDisableWarnings (State state)
+			{
+				state.WriteLine ("#pragma warning disable CS0618");
+				state.WriteLine ("#pragma warning disable CS8981");
```

Comment:
[CS8981](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves#cs8981---the-type-name-only-contains-lower-cased-ascii-characters) is presumably because we're emitting types based on filenames?  Or is that for some other reason?